src/os/config/displaysettings.js

```diff
@@ -50,6 +50,7 @@ os.config.DisplaySetting = {
   CAMERA_STATE: os.config.DisplaySettings.BASE_KEY + 'cameraState',
   CAMERA_MODE: os.config.DisplaySettings.BASE_KEY + 'cameraMode',
   MAP_MODE: os.config.DisplaySettings.BASE_KEY + 'mapMode',
+  FOG_UNSUPPORTED: os.config.DisplaySettings.BASE_KEY + 'fogUnsupported',
```

Comment:
Generally we try to use positive boolean values rather than negative ones. E.g. `fogSupported` rather than `fogUnsupported`.

Also, we generally have "supported" things as feature detections rather than configurable keys. If this was done more correctly (and that's not really on you), then the WebGL renderer would supply its own settings page.